Mobile/lib/Components/TaskDialog.dart

```diff
@@ -60,81 +60,73 @@ class _TaskdialogState extends State<Taskdialog> {
-                      style: TextStyle(fontSize: 20, color: Colors.white),
-                    ),
-                  ],
-                ),
-                content: Column(
-                crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    OutlinedText(text: widget.task.description),
-                    SizedBox(height: 10),
-                    OutlinedText(text: "Starting date: " + widget.task.startDate.toString()),
-                    SizedBox(height: 10),
-                    OutlinedText(text: "End date: " + widget.task.endDate.toString()),
-                    SizedBox(height: 10),
-                    OutlinedText(text: "Recurring: " + (widget.task.recurring ? "Yes" : "No")),
-                    SizedBox(height: 5),
-                    OutlinedText(text: "Recurring Interval: " + widget.task.recurringInterval.toString() + " day(s)"),
-                    SizedBox(height: 10),
-                    OutlinedText(text: "Single Completion: " + (widget.task.singleCompletion ? "Yes" : "No")),
+    return AlertDialog(
+      backgroundColor: CustomColorScheme.getRandomColor(),
+      scrollable: true,
+      actions: [
+        TextButton(
+          onPressed: () {
+            Navigator.of(context).pop();
+          },
+          child: OutlinedText(text: 'Close'),
+        ),
+        if (appState.user?.isParent ?? false)
+        TextButton(
+          onPressed: () => _editTask(context),
+          child: OutlinedText(text: 'Edit'),
+        ),
+      ],
 
-                    if (userAssignedToThisTask.isNotEmpty)
-                      SizedBox(height: 15),
-                      OutlinedText(text: 'Children assigned:'),
-                      for (User user in userAssignedToThisTask)
-                        Padding(
-                          padding: const EdgeInsets.only(bottom: 5, left: 5),
-                          child: OutlinedText(text: user.name),
-                        ),
-                    if (userAssignedToThisTask.isEmpty)
-                      Text('This task is not assigned to anyone.'),
-                  ],
-                ),
+      titlePadding: EdgeInsets.only(left: 20, top: 20, right: 20),
+      contentPadding: EdgeInsets.only(left: 20, top: 10, right: 20, bottom: 20),
+    
+      title: Container(
+        width: double.maxFinite,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            OutlinedText(
+              text:  widget.task.title,
+              style: TextStyle(
+                fontSize: 23,
+                color: Colors.white,
+                fontWeight: FontWeight.bold
               ),
-            );
-            }
-          ),
+            ),
+            OutlinedText(
+              text: "Reward:" + widget.task.reward.toString(),
+              style: TextStyle(fontSize: 20, color: Colors.white),
+            ),
```

Comment:
Hvorfor er task reward en del af title og ikke content?
Der mangler også et space mellem reward: og den aktuelle reward
```suggestion
            OutlinedText(
              text: "Reward: " + widget.task.reward.toString(),
              style: TextStyle(fontSize: 20, color: Colors.white),
            ),
```

Reply:
noteret også at font size på reward er forskellig fra alle andre punkter, synes de skal have den samme størrelse.